k8_cortx_cloud/start-cortx-cloud.sh

```diff
@@ -28,8 +28,8 @@ if [[ ${deployment_type} != "data-only" ]]; then
     printf "########################################################\n"
     printf "# Start CORTX Control                                   \n"
     printf "########################################################\n"
-    expected_count=1
-    kubectl scale deploy cortx-control --replicas ${expected_count} --namespace="${namespace}"
+    expected_count=$(helm get values cortx -n "${namespace}" | yq .control.replicaCount)
```

Comment:
```suggestion
    expected_count=$(helm get values cortx --all --namespace "${namespace}" --output yaml | yq .control.replicaCount)
```

This doesn't work if you install with the default value. It will return "null" in that case. You need to use `--all` to show all computed values, which includes non-user values. Without it only shows the user values.

The default output is also "table" format, which seems to look like YAML, but it has some non-YAML output. `yq` is pretty tolerable with respect to its input, but it'd be safer to explicitly output as yaml.

Reply:
Good catch.  I only tested with non-default namespace when I made this last-minute change.  I'll fix it.

Reply:
I fixed it in the most recent commit.